Libplanet/Net/Messages/ChainStatus.cs

```diff
@@ -1,36 +1,46 @@
 using System.Collections.Generic;
 using System.Numerics;
 using System.Security.Cryptography;
+using Libplanet.Blocks;
 using NetMQ;
 
 namespace Libplanet.Net.Messages
 {
-    internal class ChainStatus : Message
+    internal class ChainStatus : Message, IBlockExcerpt
```

Comment:
This looks a little bit awkward for me. Why a message inherits `IBlockExcerpt`?

Reply:
It's because this struct is used to determine the canonical chain:

https://github.com/planetarium/libplanet/blob/8f8bceb4589cefd6e5ca2304b0e88e9a0e377315/Libplanet/Net/Swarm.cs#L1338-L1344

Note that it was originally:

https://github.com/planetarium/libplanet/blob/b5fdad939dadde80ea4007e1dadc908669a21c0f/Libplanet/Net/Swarm.cs#L1337-L1342